compiler-plugin/src/main/java/io/ballerina/stdlib/log/compiler/staticcodeanalyzer/LogRule.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2025, WSO2 LLC. (http://www.wso2.org).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.log.compiler.staticcodeanalyzer;
+
+import io.ballerina.scan.Rule;
+
+import static io.ballerina.scan.RuleKind.VULNERABILITY;
+import static io.ballerina.stdlib.log.compiler.staticcodeanalyzer.RuleFactory.createRule;
+
+/**
+ * Enum to hold the log rules.
+ * @since 2.12.0
+ */
+public enum LogRule {
+    AVOID_LOGGING_CONFIGURABLE_VARIABLES(createRule(1,
+            "Potentially-sensitive configurable variables are logged", VULNERABILITY));
+    private final Rule rule;
+
+
+    LogRule(Rule rule) {
+        this.rule = rule;
+    }
+
+    public int getId() {
+        return this.rule.numericId();
+    }
+
+    public Rule getRule() {
+        return this.rule;
+    }
+
+    @Override
+    public String toString() {
+        return "{\"id\":" + this.getId() + ", \"kind\":\"" + this.rule.kind() + "\"," +
+                " \"description\" : \"" + this.rule.description() + "\"}";
+    }
+
+
```

Comment:
```suggestion
```

Reply:
Fixed in [3edb530](https://github.com/ballerina-platform/module-ballerina-log/pull/1175/commits/3edb5309e84fdca1986162244a181483a3bf3b2e)